### PR TITLE
In TaskType list, show Team name again

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open the task types view
- the names should be listed now

### Issues:
- fixes [#2875](https://github.com/scalableminds/webknossos/issues/2875)

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 - Fixed that URLs linking to a certain position in a dataset or tracing always led to the position of the active node. [#2960](https://github.com/scalableminds/webknossos/pull/2960)
 - Fixed that setting a bounding box in view mode did not work. [#3015](https://github.com/scalableminds/webknossos/pull/3015)
 - Fixed a bug where viewing Compound Annotations (such as viewing all instances of a task at once) failed with a permission issue. [#3023](https://github.com/scalableminds/webknossos/pull/3023)
+- Fixed a bug where the name was not displayed in the task types view of admins. [#3053](https://github.com/scalableminds/webknossos/pull/3053)
```
Comment (daniel-wer) [inline CHANGELOG.md]: The word "team" is missing in front of "name was not displayed ...".

File: app/assets/javascripts/admin/api_flow_types.js [hunk 2]
```diff
@@ -163,6 +163,14 @@ export type APITaskTypeType = {
   +settings: APISettingsType,
 };
 
+export type APICreateTaskType = {
```
Comment (daniel-wer) [inline app/assets/javascripts/admin/api_flow_types.js]: Let's try to avoid this type code duplication and wait for @fm3 answer :)

Comment (MichaelBuessemeyer) [reply app/assets/javascripts/admin/api_flow_types.js]: @daniel-wer ok, that was just a quick solution i came up with @philippotto. But your suggestion makes way more sense  :)

File: app/assets/javascripts/test/snapshots/public/test-bundle/test/libs/nml.spec.js.md [hunk 3]
```diff
@@ -1,126 +0,0 @@
-# Snapshot report for `public/test-bundle/test/libs/nml.spec.js`
```
Comment (daniel-wer) [inline app/assets/javascripts/test/snapshots/public/test-bundle/test/libs/nml.spec.js.md]: You accidentally deleted the NML snapshots, please restore them :)
Good thing is, this won't happen easily again in the future, thanks to @fm3s recent PR (https://github.com/scalableminds/webknossos/pull/3052) :tada:

**Pull request merged**